test/base.py

```diff
@@ -358,6 +357,13 @@ def parse_date(self, date_value):
             except ValueError:
                 continue
 
+        # Below try-catch block is used to convert the epoch to datetime value
+        # if the date_value doesn't fall under any of date formats mentioned above.
+        try:
+            return dt.utcfromtimestamp(int(date_value))
+        except ValueError:
+            LOGGER.warning(f"cannot convert {date_value} to int ")
```

Comment:
Minor suggestion, we can skip this logger statement here.

```suggestion
        try:
            return dt.utcfromtimestamp(int(date_value))
        except ValueError:
            pass
```